Statut_HSP.tex

```diff
@@ -168,19 +174,47 @@ \section{Członkowie Stowarzyszenia}
   regularnego opłacania składek.
 \end{enumerate}
 
-\myparagraph{§ 14}
+\myparagraph{§ 12.NZ.1}
+\begin{enumerate}
+\item
+  Prawa i obowiązki każdego członka nadzwyczajnego z osobna definiuje zarząd.
```

Comment:
Może warto zaznaczyć, że Zarząd definiuje w uchwale o nadaniu statusu członka nadzwyczajnego.

Reply:
To jest w 11.4